### ISSUE TITLE
ARIA improvements for list view

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#6525]
This ticket offers some concrete solutions for the issue posed in #4240.

[List view](https://fullcalendar.io/docs/list-view) is essentially a multi-header table. The title for each day is a "header" *within* the body of the table. [The w3c website](https://www.w3.org/WAI/tutorials/tables/multi-level/#table-with-three-headers-related-to-each-data-cell) shows a nice way to architect this in an ARIA-friendly way.

Aside from defining a relationship between cells, list view will also benefit from the following ARIA-related changes:

- the cell that holds the little dot (`fc-list-graphic`) should be marked aria-hidden="true" (as mentioned in https://github.com/fullcalendar/fullcalendar/issues/2536#issuecomment-765508667)
- better semantics within each day header for text like "Sunday" and "August 29, 2021", for screen readers

Proposed HTML skeleton:

```html
<table>
  <thead><!-- TODO: somehow hide the thead for non-screen readers -->
    <tr>
      <th scope="col" id="time-header">Time</th>
      <th scope="col" aria-hidden="true"><!-- color dots --></th>
      <th scope="col" id="even-title-header">Event Title</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th id="date-header" colspan="3" scope="colgroup" aria-labeledby="short-date" aria-describedby="long-date">
        <div id="long-date" style="float:right">August 29, 2021</div>
        <div id="short-date">Sunday</div>
      </th>
    </tr>
    <tr>
      <td headers="time-header date-header">
        10:30am
      </td>
      <td aria-hidden="true">
        <!-- color dot markup -->
      </td>
      <td headers="event-title-header date-header">
        Conference
      </td>
    </tr>
  </tbody>
</table>
```

The thead is only useful for screen readers and should be hidden for non-screen reading devices. [Here are some ideas](https://webaim.org/techniques/css/invisiblecontent/) on how to hide content.

**Comment by arshaw** [comment]
The mentioned improvements were implemented and released in [v5.10.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v5.10.0)

**Comment by room34** [comment]
Concerning this bit:

`<th scope="col" aria-hidden="true"><!-- color dots --></th>`

The `<!-- color dots -->` comment is not appearing in the output, and I'm not sure if that would count as cell content anyway. The issue is that this empty `<th>` tag is causing the WAVE tool to report a **Empty table header** error. Their recommended fix is to use `<td>` tags instead.

I investigated whether or not `<td>` tags are strictly allowed inside `<thead>` and at least according to this argument presented [https://stackoverflow.com/questions/45926582/is-td-allowed-inside-thead](on Stack Overflow), they are.
